### PR TITLE
Added ajax support.

### PR DESCRIPTION
**Pull request by milkovsky**

### CODE REVIEW

File: src/Plugin/views/argument_default/EntityRenderHistory.php [hunk 1]
```diff
@@ -53,6 +53,19 @@ public function getEntityTypeManager() {
...
 
+  /**
+   * Retrieves the currently active request object.
+   *
+   * @return \Symfony\Component\HttpFoundation\Request
+   *   The currently active request object.
+   */
+  public function getCurrentRequest() {
+    if (empty($this->entityTypeManager)) {
+      $this->entityTypeManager = \Drupal::request();
```
Comment (fago) [inline src/Plugin/views/argument_default/EntityRenderHistory.php]: ouch.

File: src/Plugin/views/argument_default/EntityRenderHistory.php [hunk 2]
```diff
@@ -85,6 +98,14 @@ public function getArgument() {
     $ids = $this->getEntityRenderHistory()
       ->getRenderedEntities($this->options['entity_type_id']);
 
+    if (!$ids) {
```
Comment (fago) [inline src/Plugin/views/argument_default/EntityRenderHistory.php]: I think we would have to merge the ids here.

**Review by fago — CHANGES_REQUESTED** [review]

File: src/Plugin/views/argument_default/EntityRenderHistory.php (hunk 1 above)
Comment (milkovsky) [reply src/Plugin/views/argument_default/EntityRenderHistory.php]: fixed

File: src/Plugin/views/argument_default/EntityRenderHistory.php (hunk 2 above)
Comment (milkovsky) [reply src/Plugin/views/argument_default/EntityRenderHistory.php]: done

**Pull request closed without merging**